src/services/dashboards/widgets/_components/WidgetDataTable.vue

```diff
@@ -80,10 +80,12 @@
                                           v-bind="getColSlotProps(item, field, colIndex, rowIndex)"
                                     >
                                         <p-tooltip position="bottom"
-                                                   :contents="isEllipsisActive(rowIndex, colIndex) ? getTooltipContents(item, field) : undefined"
+                                                   :contents="isEllipsisActive(rowIndex, colIndex, fields[colIndex]) ? getTooltipContents(item, field) : undefined"
```

Comment:
question: Isn't it `fields` and not `props.fields`? If `props.fields`, it doesn't seem necessary to be in the argument.